### PR TITLE
Document Animal data access

### LINKED ISSUE #13: Document animal data access
Discuss here.

- [ ] do we automate brukraw conversion?

### PR DESCRIPTION
**Pull request by jeremie-fouquet**
This pull request partly resolves #13.

### CODE REVIEW

File: docs/doing_things/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Doing Things
 
-## Accessing the System
+## <a name="systemAccess"></a>Accessing the System
```
Comment (gdevenyi) [inline docs/doing_things/index.md:3]: There's no need for adding anchors, they're auto-generated by sphinx.

File: docs/doing_things/index.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ a graphical display of the current state of CPU and Memory utilization.
 Remote access from outside the Douglas is provided by authorization only.
 Please contact the platform to request access.
 
-## Transferring Data
+## <a name="dataTransfer"></a>Transferring Data
```
Comment (gdevenyi) [inline docs/doing_things/index.md:35]: Ditto

File: docs/doing_things/index.md [hunk 3]
```diff
@@ -56,6 +56,23 @@ to long-term cold storage. Users are expected to access their data during that
...
+days from the date of the acquisition at `/opt/<PV version>/data/<username>`, 
+with `<PV version>` the ParaVision software version (PV5.1 or PV6.0.1) and `<username>` 
+the login of the user performing the scans. Users are expected to access their data 
+during that time and copy it to appropriate long term storage. Recovery is available 
+with delayed access and a recovery fee associated with staff time. Data can be accessed
+by 1) accessing the [Neuroinformatics Platform](#systemAccess) and then 2) 
+accessing the bruker7t computer. The [tools for data transfer](#dataTransfer) will 
+work, provided you take into account you need to connect to an additional computer (the bruker7t).
+
+Conversion to other formats (dicom and nifti) is available upon request.
```
Comment (gdevenyi) [inline docs/doing_things/index.md:73]: DICOM and NIFTI

File: docs/doing_things/index.md [hunk 4]
```diff
@@ -56,6 +56,23 @@ to long-term cold storage. Users are expected to access their data during that
...
+
+The Bruker BioSpec 70/30 scanner produces data in the raw Bruker data format. 
+The data sets are accessible on the Bruker controlling computer for 14 calendar
+days from the date of the acquisition at `/opt/<PV version>/data/<username>`, 
+with `<PV version>` the ParaVision software version (PV5.1 or PV6.0.1) and `<username>` 
+the login of the user performing the scans. Users are expected to access their data 
+during that time and copy it to appropriate long term storage. Recovery is available 
+with delayed access and a recovery fee associated with staff time. Data can be accessed
+by 1) accessing the [Neuroinformatics Platform](#systemAccess) and then 2) 
+accessing the bruker7t computer. The [tools for data transfer](#dataTransfer) will
```
Comment (gdevenyi) [inline docs/doing_things/index.md:70]: You can link to the auto-generated section anchors like the example here
https://github.com/DouglasNeuroInformatics/Douglas_User_Documentation/blob/master/docs/about_the_platform/index.md?plain=1#L28

Doc ref:
https://myst-parser.readthedocs.io/en/v0.15.1/sphinx/use.html#automatically-create-targets-for-section-headers

**Review by gdevenyi — CHANGES_REQUESTED** [review]

**Comment by gdevenyi** [comment]
Cheers thanks!

**Comment by surchs** [comment]
@all-contributors please add @jeremie-fouquet for content, doc

**Pull request merged**